test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -498,6 +498,16 @@ def test_interpolate_upsample(self):
     def test_interpolate_downsample(self):
         self._interpolate_tests(False)
 
+    def test_groupnorm(self):
```

Comment:
More thorough end to end tests please

Reply:
Are you asking for different channel/num group cases?

Reply:
I addressed the two comments about the logic and commented in the code. Please let me know
which part is missing or not covered in the tests.

Reply:
You can add the below additional test cases https://pytorch.org/docs/master/nn.html#groupnorm:

> \# Separate 6 channels into 6 groups (equivalent with InstanceNorm)
> m = nn.GroupNorm(6, 6)
> \# Put all 6 channels into a single group (equivalent with LayerNorm)
> m = nn.GroupNorm(1, 6)

To test https://github.com/pytorch/pytorch/blob/aea140523a5c7fdf017439fbcc461cafd95c66d4/torch/onnx/symbolic_opset9.py#L1987
try construct test case with `weight`, `bias` being None

Reply:
Thanks @BowenBao. I added the tests for various num_groups.
I just verified case where weight/bias is none is covered in test_groupnorm_noaffine test.